RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContentCosmics_cff.py

```diff
@@ -27,9 +27,13 @@
                                            'keep HBHERecHitsSorted_hbherecoMB_*_*',
                                            'keep HORecHitsSorted_horecoMB_*_*',
                                            'keep HFRecHitsSorted_hfrecoMB_*_*',
-                                           'keep ZDCDataFramesSorted_*Digis_*_*',
+                                           #'keep ZDCDataFramesSorted_*Digis_*_*',
+                                           'keep ZDCDataFramesSorted_hcalDigis_*_*',
+                                           'keep ZDCDataFramesSorted_simHcalUnsuppressedDigis_*_*',
                                            'keep ZDCRecHitsSorted_*_*_*',
-                                           'keep HcalUnpackerReport_*_*_*'
+                                           #'keep HcalUnpackerReport_*_*_*'
+                                           'keep HcalUnpackerReport_castorDigis_*_*'
```

Comment:
there is a comma missing here.
@jeongeun please double-check all other files and update only after ```runTheMatrix.py -l limited -i all``` completes